docs/make.jl

```diff
@@ -0,0 +1,19 @@
+using Documenter, AnalyzeRegistry
+
+DocMeta.setdocmeta!(AnalyzeRegistry, :DocTestSetup, :(using AnalyzeRegistry); recursive=true)
+
+makedocs(
+    format = Documenter.HTML(
+        prettyurls = true
+    ),
+    modules = [AnalyzeRegistry],
+    sitename = "AnalyzeRegistry.jl",
+    pages = ["Home" => "index.md",
+             "API Reference" => "api.md",
+             "Saving results" => "serialization.md"]
+)
+
+deploydocs(
+    repo = "https://github.com/giordano/AnalyzeRegistry.jl",
+    push_preview=true,
```

Comment:
we'll get per-PR doc previews this way (which is why I made the PR from your repo instead of my fork, so that we can get previews for this PR)